.travis.yml

```diff
@@ -48,7 +48,53 @@ matrix:
       env: TOXENV=py35-django22
     - python: 3.5
       env: TOXENV=py35-django21
+      # Adding jobs for ppc64le
+    - python: 3.7
+      arch: ppc64le
+      env: TOXENV=py37-flake8
+    - python: 3.7
+      arch: ppc64le
+      env: TOXENV=py37-docs
+
+    - python: 3.8
+      arch: ppc64le
+      env: TOXENV=py38-django30
+    - python: 3.8
+      arch: ppc64le
+      env: TOXENV=py38-django22
+    - python: 3.8
+      arch: ppc64le
+      env: TOXENV=py38-django21
+    - python: 3.8
+      arch: ppc64le
+      env: TOXENV=py38-djangomaster
 
+    - python: 3.7
+      arch: ppc64le
+      env: TOXENV=py37-django30
+    - python: 3.7
+      arch: ppc64le
+      env: TOXENV=py37-django22
+    - python: 3.7
+      arch: ppc64le
+      env: TOXENV=py37-django21
+    - python: 3.7
+      arch: ppc64le
+      env: TOXENV=py37-djangomaster
+
+    - python: 3.6
+      arch: ppc64le
+      env: TOXENV=py36-django22
+    - python: 3.6
+      arch: ppc64le
+      env: TOXENV=py36-django21
+
+    - python: 3.5
```

Comment:
I think python 3.5 can be dropped

Reply:
Shall I drop python 3.5 jobs for all the architectures or for ppc64le architecture alone?

Reply:
Jobs for Python 3.5 been dropped for intel as well as ppc64le architectures. Thanks!